### ISSUE TITLE
SB2 (13") surface performance : device not found

### ISSUE THREAD

**Issue by deb75** [linux-surface/surface-control#9]
When I issue : 
```
surface performance get
```
I receive
```
Error : cannot access device
Surface performance-mode device not found
```

In `/sys/devices/platform/MSHW0107\:00` I just have : 
```
driver_override  modalias   uevent driver                    firmware_node    subsystem
```

But all `surface_*` modules are currently loaded in the surface kernel 4.19.146

Am I missing something ?

Regards

**Comment by qzed** [comment]
The device ID for the performance mode has changed and due to some backporting issues with v4.19 and v5.4 it didn't get updated correctly. Should be fixed with the next release (v4.19.146-3).

**Comment by qzed** [comment]
Fixed with the latest kernel release. Please make sure to update both the kernel and the surface-control packages if the issue still occurs.
